stable/atlantis/README.md

```diff
@@ -1,29 +1,36 @@
 # Atlantis
+
 [Atlantis](https://www.runatlantis.io/) is a tool for safe collaboration on [Terraform](https://www.terraform.io/) repositories.
 
 ## Introduction
-This chart creates a single pod in a StatefulSet running Atlantis.  Atlantis persists Terraform [plan files](https://www.terraform.io/docs/commands/plan.html) and [lock files](https://www.terraform.io/docs/state/locking.html) to disk for the duration of a Pull/Merge Request.  These files are stored in a PersistentVolumeClaim to survive Pod failures.
+
+This chart creates a single pod in a StatefulSet running Atlantis. Atlantis persists Terraform [plan files](https://www.terraform.io/docs/commands/plan.html) and [lock files](https://www.terraform.io/docs/state/locking.html) to disk for the duration of a Pull/Merge Request. These files are stored in a PersistentVolumeClaim to survive Pod failures.
 
 ## Prerequisites
--   Kubernetes 1.9+
--   PersistentVolume support
+
+- Kubernetes 1.9+
+- PersistentVolume support
 
 ## Required Configuration
+
 In order for Atlantis to start and run successfully:
+
 1. At least one of the following sets of credentials must be defined:
-    -  `github`
-    -  `gitlab`
-    -  `bitbucket`
 
-    Refer to [values.yaml](values.yaml) for detailed examples.
+   - `github`
+   - `gitlab`
+   - `bitbucket`
+
+   Refer to [values.yaml](values.yaml) for detailed examples.
 
 1. Supply a value for `orgWhitelist`, e.g. `github.org/myorg/*`.
 
 ## Customization
-The following options are supported.  See [values.yaml](values.yaml) for more detailed documentation and examples:
+
+The following options are supported. See [values.yaml](values.yaml) for more detailed documentation and examples:
 
 | Parameter                                   | Description                                                                                                                                                                                                                                                                                               | Default |
-|---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| ------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
```

Comment:
Why add the space? This website I usually use to format doesn't do that: http://markdowntable.com/